server/utils/validation/signUpCheckInput.ts

```diff
@@ -0,0 +1,14 @@
+import Joi from 'joi';
+
+const signUpCheckInput = async (data: object) => {
```

Comment:
Just rename it `signupValidation` so you all have the same style of naming and the same for the file name